tests/unittests/MemoryAllocatorTestModels.txt

```diff
@@ -0,0 +1,8254 @@
+MODEL cifar10_quant.tflite
```

Comment:
It would be nice to describe somewhere in the docs or source comments how this file was produced and how to update it if e.g. a model changes or a new model needs to be added.

Reply:
Added a header within this file about how it was obtained and how somebody can add manually another model:
```
################################################################################
# This file describes a sequence of activation memory allocation/deallocation
# instructions for multiple models. This file is used to test the function
# "allocateAll()" from the MemoryAllocator object. This file was generated
# automatically using a Matlab script which parses the Glow IR obtained during
# Glow compilation with the option -dump-ir for all the models. This text file
# has a <key, value(s)> format with the following convention:
#   NAME              - Model name (used as delimiter between models).
#   ALIGN <ALIGNMENT> - Alignment (in bytes) used for memory allocator.
#   ALLOC <ID> <SIZE> - Instruction to allocate a buffer with identifier <ID>
#                       and size <SIZE>. Each buffer must have a unique ID.
#   FREE <ID>         - Instruction to free a previously allocated buffer with
#                       identifier <ID>. For a given model every buffer which
#                       is allocated must be later freed.
#   MEM <SIZE>        - Expected memory usage for the alloc/free sequence.
#   EFF <RATIO>       - Expected efficiency of the allocation
# You can define manually additional unit tests following the given convention.
################################################################################
```